src/components/SignInForm.jsx

```diff
@@ -117,95 +118,101 @@ const SignIn = (props, routeProps) => {
-                  <ErrorMsg
-                    errormessage={errors.Email}
-                    touched={touched.Email}
+                  <p role="alert" className="error-message">
+                    <ErrorMsg
+                      errormessage={errors.Email}
+                      touched={touched.Email}
+                    />
+                  </p>
+                </div>
+                <div
+                  className={
+                    props.errors.Password && props.touched.Password
+                      ? "cs-form-control error"
+                      : "cs-form-control"
+                  }
+                >
+                  <label for="PasswordLabel" class="dg_label">
+                    <span class="dg_label-text">
+                      {SignInPage.PasswordLabel}
+                    </span>
+                  </label>
+                  <Field
+                    type={fieldType === "Password" ? "Password" : "text"}
+                    name="Password"
+                    className={`text-input ${
+                      errors.Password && touched.Password ? "error" : ""
+                    }`}
                   />
-                </p>
-              </div>
-              <div
-                className={
-                  props.errors.Password && props.touched.Password
-                    ? "cs-form-control error"
-                    : "cs-form-control"
-                }
-              >
-                <label name="Password" htmlFor="password">
-                  {SignInPage.PasswordLabel}
-                </label>
-                <Field
-                  type={fieldType === "Password" ? "Password" : "text"}
-                  name="Password"
-                  className={`text-input ${
-                    errors.Password && touched.Password ? "error" : ""
-                  }`}
-                />
-                <span
-                  onClick={handlePasswordToggleChange}
-                  className={`fa fa-fw fa-eye field-icon ${
-                    fieldType === "text" ? "fa-eye-slash" : ""
-                  }`}
-                ></span>
+                  <span
+                    onClick={handlePasswordToggleChange}
+                    className={`fa fa-fw fa-eye field-icon ${
+                      fieldType === "text" ? "fa-eye-slash" : ""
+                    }`}
+                  ></span>
 
-                <p role="alert" className="error-message">
-                  <ErrorMsg
-                    errormessage={errors.Password}
-                    touched={touched.Password}
+                  <p role="alert" className="error-message">
+                    <ErrorMsg
+                      errormessage={errors.Password}
+                      touched={touched.Password}
+                    />
+                  </p>
+                </div>
+                <div className="cs-form-control">
+                  <p htmlFor="forgetpassword">
+                    {" "}
```

Comment:
Seem's like we don't need this?

Reply:
I did not work on this one. @tmccoy529 , did u work on this one?

Reply:
removed